common/config/rush/browser-approved-packages.json

```diff
@@ -1065,6 +1065,10 @@
     {
       "name": "yup",
       "allowedCategories": [ "examples" ]
+    },
+    {
+      "name": "copy-to-clipboard",
+      "allowedCategories": [ "examples", "production", "tools" ]

```

Comment:
Did you add this entry manually? Do we need this in all the categories? Rush update should add this entry automatically with the correct categories and correctly sorted.

Reply:
fixed